plone/app/event/tests/test_portlet_calendar.py

```diff
@@ -275,53 +269,61 @@ def test_with_collection_as_source(self):
         are used to build the calendar
         """
         tz = pytz.timezone(TZNAME)
-        start = tz.localize(datetime.now())
-        end = start + timedelta(hours=1)
 
-        createContentInContainer(self.portal, PTYPE, title="e1", start=start, end=end)
-
-        createContentInContainer(self.portal, PTYPE, title="e2", start=start, end=end)
+        start = tz.localize(datetime.now())
+        # Set roughly in the mid of the month to avoid issues when testing at
+        # the end of a month where the tomorrow dates are already out of the
+        # range of the current calendar month.
+        start = start.replace(day=15)
```

Comment:
The way this is written could imply that there is a bug in the package, and not in the test. At least I think the issue is with only the test... 🤔 

Also I think the condition exists at the start of the month, as well as the end, since it checks yesterday, too.

```suggestion
        # Testing at the start or end of the month is prone to a month
        # rollover condition, which will cause a test failure. This is not
        # a bug in plone.app.event, but merely a test condition failure.
        # To avoid this issue, set the start date to roughly in the middle
        # of the month.
        start = start.replace(day=15)
```